.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -525,3 +551,34 @@ void updateAntoraYaml(String directory) {
         echo 'No updateAntoraYaml due to testing build'
     }
 }
+
+/* OptaPlanner Operator */
+
+String getOperatorImageRegistry() {
+    return params.OPERATOR_IMAGE_REGISTRY
+}
+
+String getOperatorImageNamespace() {
+    return params.OPERATOR_IMAGE_NAMESPACE
+}
+
+String getOperatorImageName() {
+    return env.OPERATOR_IMAGE_NAME
+}
+
+String getOperatorImageTag() {
+    return params.OPERATOR_IMAGE_TAG ?: getProjectVersion() ?: 'latest'
```

Comment:
This means for now in Optaplanner nightly, it will upload the image with `latest` right ?

Reply:
No, we don't push the image in nightlies; we just build it locally (and later use it in integration tests). I will remove the 'latest' as a fallback option.

Reply:
yeah sorry, missed the `isRelease` filter ;)